web/src/main/java/org/mskcc/cbio/portal/web/ProxySessionServiceController.java

```diff
@@ -13,41 +13,34 @@
 
 import javax.servlet.http.HttpServletResponse;
 
-import org.apache.commons.lang.StringUtils;
 import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.lang.StringUtils;
+import org.cbioportal.web.parameter.VirtualStudy;
+import org.cbioportal.web.parameter.VirtualStudyData;
 import org.json.simple.JSONObject;
-import org.mskcc.cbio.portal.model.virtualstudy.VirtualStudy;
```

Comment:
What about these models?

Reply:
well, that from `business` module and `StudyViewFilter` which is in `web` module is not accessible there. So I had to create new ones in `web` module.

classes in `business` module can be deleted once the legacy servlet api's are cleaned/deleted

Reply:
I don't think it is a good idea to create actual models under `web/parameter`, we should create them under `model` module. Those `parameter` classes only to be used to define user input. Also, as far as I know `business` module is a legacy module.

If we really need to use study view filter in our model, probably we should move something from web to model, not model to web.

Reply:
@onursumer `VirtualStudy` is not actually a cBioPortal database model. Not sure if we want to have that in `model` module. And moving `StudyViewFilter` parameter class to `module` would actually need to move around 5 dependent classes - not sure if we want that